### PR TITLE
Modifying README file

### PR DESCRIPTION
**Pull request by aabeshov**
### Description
Instructions for building ci-images have been updated to work correctly.

### Issues Resolved
[#3478 ]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -10,8 +10,11 @@ We build, assemble, and test our artifacts on docker containers. All of our pipe
 
 If you only want to build the docker image for either x64 or arm64, run this on a x64 or arm64 host respectively:
 
+You need to run the script within the `opensearch-build/docker/ci` folder. Running it
```
Comment (dblock) [inline docker/README.md:13]: Don't fold at 80 chars? Maybe merge with the sentence above:

"To build the docker image for either x64 or arm64, run `docker` on a x64 or arm64 host respectively within the `pensearch-build/docker/ci` folder:"

Comment (aabeshov) [reply docker/README.md]: > Don't fold at 80 chars? Maybe merge with the sentence above:
> 
> "To build the docker image for either x64 or arm64, run `docker` on a x64 or arm64 host respectively within the `pensearch-build/docker/ci` folder:"

Yeap sure, edited instruction.

File: docker/README.md [hunk 2]
```diff
@@ -8,10 +8,10 @@ We build, assemble, and test our artifacts on docker containers. All of our pipe
 
 ### Build CI Runner Docker Image from Dockerfile
 
-If you only want to build the docker image for either x64 or arm64, run this on a x64 or arm64 host respectively:
+To build the docker image for either x64 or arm64, run docker on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:
```
Comment (gaiksaya) [inline docker/README.md:11]:
```suggestion
To build the docker image for either x64 or arm64, run the below command on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:
```

Comment (aabeshov) [reply docker/README.md]: > Suggested change
> To build the docker image for either x64 or arm64, run docker on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:
> To build the docker image for either x64 or arm64, run the below command on a x64 or arm64 host respectively within the `opensearch-build/docker/ci` folder:

Done,could you check it please when you have free time?

**Review by gaiksaya — APPROVED** [review]

**Comment by prudhvigodithi** [comment]
Hey @aabeshov can you please rebase from upstream?

**Comment by aabeshov** [comment]
> Hey @aabeshov can you please rebase from upstream?

Oh yeah sorry, forgot it.

**Pull request merged**